### ISSUE TITLE
Accessability for Tabs - aria-selected="true" is preferred instead of "page" 

### ISSUE THREAD

**Issue by s1x** [tailwindlabs/tailwindui-issues#591]
This is not a bug but an improvement for accessibility.

**What component (if applicable)**
- URL for category: https://tailwindui.com/components/application-ui/navigation/tabs
- Component name: Tabs

**Describe the bug**
This is not really a bug, but an improvement for accessibility for tabs.
Instead of aria-selected="page" -> "true" should be used.

**Additional context**
https://www.w3.org/TR/wai-aria-1.1/#tablist

**Comment by davidluhr** [comment]
@s1x Thanks for the feedback.

Please note that our current "Tabs" aren't interactive tabs for showing/hiding content within the same page, they are simply a visual presentation of a link navigation. This approach is accessible and valid, as it's simply a group of links with no special functionality. The expected behavior is to show a new page when clicking on one of these links.As such, `aria-current="page"` is accurate for notifying assistive technology that one of the links points to the current page.

It's possible that we will be updating these examples at some point to include interactive tab components, which would show/hide content on the same page, and would use the appropriate ARIA for that use case (`role="tab"`, role="tabpanel"`, `aria-selected="true"`, etc.), along with the needed keyboard support.

At this time, no change is needed, as the current components are not actually Tabs in an accessibility sense.

**Comment by s1x** [comment]
Hey @davidluhr,
many thanks for taking the time and the clarification. Reading this I think you're absolutely right. 👍
